File: 1295-find-numbers-with-even-nubmers-of-digits/1295.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        count = 0
        for num in nums:
            digit_count = 0
            while num:
                digit_count += 1
                num //= 10
            if digit_count % 2 == 0:
                count += 1
        return count

File: 1295-find-numbers-with-even-nubmers-of-digits/1295.rs
impl Solution {
    pub fn find_numbers(nums: Vec<i32>) -> i32 {
        let (mut count, mut digit_count) = (0, 0);

        for mut num in nums {
            digit_count = 0;

            while num != 0 {
                digit_count += 1;
                num /= 10;
            }

            if digit_count % 2 == 0 {
                count += 1
            }
        }
        count 
    }
}
